app/src/components/constants.js

```diff
@@ -21,6 +21,9 @@ module.exports = Object.freeze({
     NONE: 'NONE'
   },
 
+  /** Need to specify valid AWS region or it'll explode ('us-east-1' is default, 'ca-central-1' for Canada) */
+  DEFAULTREGION: 'us-east-1',
```

Comment:
Wondering do we want to have this defined in config file instead? As in utils/getBucket the rest of the config values are being read from the config file.

Reply:
That might be a good idea for other people hosting their own COMS down the line yeah indeed.
For this just moving the constant for use in more than 1 spot but think that's worth making a backlog item for investigating at least.

Reply:
The full intent of COMS is that it will eventually (if not already doing so) accept a user-specified region. This should only be in config IF and ONLY IF the bucket must specify its precise region for the default bucket scenario; otherwise this should default to the AWS default of `us-east-1`, which is why it's treated more as an in-code constant.

Reply:
Yeah, worth tracking, agree the call specifying it will be useful when that gets implemented (other comment in here).
If anyone is using the default bucket mode that would be a good place to take it into consideration for config.